docsrc/imap/download/release-notes/index.rst

```diff
@@ -28,112 +22,86 @@ Latest development snapshot is |imap_development_release_notes|. Documentation a
     there will generally be **large breaking changes** occurring between
     releases in this series.
 
-Supported Product Series
-========================
+Series 3.14
+~~~~~~~~~~~
+
+.. toctree::
+    :maxdepth: 1
+
+    3.13/index
+
+Supported Verions
+-----------------
+
+The latest stable version is |imap_stable_release_notes|.  Its documentation
+can be found at :cyrus-stable:`/`.
 
 Series 3.12
------------
+~~~~~~~~~~~
```

Comment:
This section of changes is sort of awkward.  The master branch is currently in a state of lying that 3.12 is the current stable version, because its own version is already 3.13.  Whereas the main website that's built from the 3.10 branch doesn't even know 3.12 exists yet.  Once 3.12 is released for reals, the lies will become true.  But in the meantime, landing and backporting this set of changes is a little awkward.

If we don't intend to backport these changes (or only as far as 3.12, so that the main website only sees the updates once 3.12 is released), then it's less awkward.

Reply:
Yeah, I'd say that it was broken/awkward/wrong when I got here, and I've just reorganized things.  I think we should fix this, but I don't think this branch is making things worse right now.  Also, **I don't think we need to backport these changes**.